src/components/InvertedFlatList/BaseInvertedFlatList.js

```diff
@@ -29,19 +28,14 @@ const defaultProps = {
     shouldMeasureItems: false,
 };
 
-class BaseInvertedFlatList extends Component {
-    constructor(props) {
-        super(props);
+function BaseInvertedFlatList(props) {
```

Comment:
```suggestion
function BaseInvertedFlatList({initialRowHeight, shouldMeasureItems, innerRef, renderItem}) {
```

Reply:
Hi @gedu, I'm kind of aware of the props destructuring, but I had to leave it there for this case because of props forwarding and the forward ref that is used in this component.